kerlescan/config.py

```diff
@@ -12,3 +12,5 @@
 prometheus_multiproc_dir = os.getenv("prometheus_multiproc_dir", None)
 
 path_prefix = os.getenv("PATH_PREFIX", "/api/")
+
+bypass_rbac = os.getenv("BYPASS_RBAC", None)
```

Comment:
I tend to think of feature flags as boolean enable/disable of functionality, something akin to `ENABLE_RBAC`. For me, this presents this more in a security context, which I understand. However, in production, where we want to use this for a period of time we aren't bypassing but rather disabling the feature, is that correct?

Reply:
If the flag is set to the string "BYPASS", the shared method is still called by the services but it just short-circuits and returns immediately, without calling the rbac service. I tried to write it to be non-invasive since it's temporary code.

I tried to stick to the same variable name as https://github.com/RedHatInsights/e2e-deploy/blob/master/templates/catalog/catalog-api.yml#L86-L89 so there weren't two ways to say the same thing. They use true/false, but I prefer using the string. IMO the string is clearer if you are having to make it work at 2am.

Reply:
I can live with this. But I am not a fan. I would prefer if we standardize on true feature flags, e.g. `ENABLE_RBAC` with the value set to true in environments we want it turned on. Just because this is security related, shouldn't empower a different set of nomenclature for feature enablement. Ultimately, RBAC is still a feature. In my view, we either want it enabled or disabled. 

To your point, at 2am, I will have to dig through code to know to pass 'BYPASS' vs. setting true / false on the deployment configuration. (Granted CI/CD should overwrite anything we set on the deployment config anyway).

Reply:
I'm 2 for 2 for causing confusion :)

I'll close this and re-open w/ ENABLE_RBAC.